Cargo.toml

```
@@ -16,16 +16,18 @@ commands = []
 system_param = []
 
 [dependencies]
-bevy = { version = "0.14", default-features = false }
-noisy_bevy = "0.7"
+bevy = { version = "0.15", default-features = false }
+noisy_bevy = "0.8.0"
 
 [dev-dependencies]
-bevy = { version = "0.14", default-features = false, features = [
-  "bevy_winit",
-  "bevy_sprite",
-  "webgl2",
-  "x11",
+bevy = { version = "0.15", default-features = false, features = [
+    "bevy_winit",
+    "bevy_sprite",
+    "webgl2",
+    "x11",
```

Bevy and most other crates I know does 2 space indentation for toml files, let's keep the convention.

```suggestion
    "bevy_winit",
    "bevy_sprite",
    "webgl2",
    "x11",
```
```suggestion
  "bevy_winit",
  "bevy_sprite",
  "webgl2",
  "x11",
```